src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -27,6 +27,7 @@ module sonic-portchannel {
 	}
 
 	container sonic-portchannel {
+
```

Comment:
Add PORTCHANNEL_INTERFACE yang as well

Reply:
is this todo list? I just saw another pr about portchannel_interface.

Reply:
https://github.com/Azure/sonic-buildimage/pull/7034

Reply:
yes it was todo, I wanted to make sure PORTCHANNEL_INTERFACE is also done soon.